.github/workflows/ci.yaml

```diff
@@ -7,18 +7,18 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v5
         with:
-          go-version: 1.23.x
+          go-version: 1.25.x
       - name: Checkout code
         uses: actions/checkout@v4
       - name: Run linters
-        uses: golangci/golangci-lint-action@v5
+        uses: golangci/golangci-lint-action@v8
         with:
           version: latest
           args: --timeout=3m
   go-test:
     strategy:
       matrix:
-        go-version: [1.23.x]
+        go-version: [1.25.x]
```

Comment:
can we please take this out of `matrix` and use `go-version-file: go.mod` please like so:

https://github.com/ConductorOne/baton-google-cloud-platform/blob/6128297387a21a90cb7e479c75c28ff7c0e47e1a/.github/workflows/ci.yaml#L12

It has to be done after the go install step